be/src/exprs/dict_query_expr.cpp

```diff
@@ -107,6 +107,11 @@ Status DictQueryExpr::open(RuntimeState* state, ExprContext* context, FunctionCo
     // init parent open
     RETURN_IF_ERROR(Expr::open(state, context, scope));
 
+    // make sure ExprContext::clone will not open DictQueryExpr again
```

Comment:
Doesn't prepare/close need to deal with scope?

Reply:
`prepare` can not be call multiple time in current expression framework and `dictQueryExpr::close` is safe to called multiple time without any change. I think it is ok.